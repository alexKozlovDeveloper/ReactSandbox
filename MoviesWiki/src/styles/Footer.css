.footer {
    bottom: 0;
    width: 100%;
    background: #333;
    padding: 10px 0;
    color: #fff;
    min-width: 500px;
}

.container {
    width: 80%;
    margin: 0 auto;
    text-align: center;
}