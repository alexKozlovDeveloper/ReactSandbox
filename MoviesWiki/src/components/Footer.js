import React, { Component } from "react";

import styles from "../styles/Footer.css"

const Footer = ({ config }) => {
  return <div className={styles.footer}>
    <div className={styles.container}>{config.title}</div>
  </div>;
}

export default Footer;