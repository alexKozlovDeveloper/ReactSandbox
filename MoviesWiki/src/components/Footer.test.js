import React from 'react';
import { shallow, configure } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import Footer from './Footer';

configure({ adapter: new Adapter() });

test('Footer rendering test', () => {
  const footerConfig = {
    title: 'Copyright Â© 2019 AkTest',
  };

  const component = shallow(<Footer config={footerConfig} />);
  expect(shallowToJson(component)).toMatchSnapshot();
});
