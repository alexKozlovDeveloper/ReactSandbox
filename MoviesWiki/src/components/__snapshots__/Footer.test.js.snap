// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer rendering test 1`] = `
<div
  className="footer"
>
  <div
    className="container"
  >
    Copyright Â© 2019 AkTest
  </div>
</div>
`;
